ruma-events-macros/src/lib.rs

```diff
@@ -145,3 +148,13 @@ pub fn derive_from_raw(input: TokenStream) -> TokenStream {
         .unwrap_or_else(|err| err.to_compile_error())
         .into()
 }
+
+/// Generates an implementation of `ruma_events::StateEventContent`. This trait allows event
+/// content types to be used inside of `StateEvent<C: StateEventContent>`.
```

Comment:
I think the trait's purpose should be described in the trait's docs, not in the derive macro's docs.

```suggestion
/// Generates an implementation of `ruma_events::StateEventContent`.
```

Reply:
Also, maybe mention `RoomEventContent` as well?